cognite/pygen/_query/interface.py

```diff
@@ -28,15 +28,23 @@ class QueryExecutor:
         client (CogniteClient): An instance of the CogniteClient.
         views (Sequence[dm.View], optional): A list of views to use for the queries. Defaults to None.
             If not passed, the views will be fetched from the server when needed.
+        unpack_edges (Literal["skip", "include"], optional): Whether to unpack edges in the result.
+            If "skip", edges will not be included in the result. If "include", edges will be included.
+            Defaults to "include".
     """
 
-    def __init__(self, client: CogniteClient, views: Sequence[dm.View] | None = None):
+    def __init__(
+        self,
+        client: CogniteClient,
+        views: Sequence[dm.View] | None = None,
+        unpack_edges: Literal["skip", "include"] = "skip",
```

Comment:
In agreement with the Atlas AI team, change the default behavior to not unpack edges.

Reply:
> Is this a breaking change for other users of PyGen? I am not sure about the user base, but "include" was the previous default value.